pbkdf2/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pbkdf2"
-version = "0.11.0"
+version = "0.12.0-pre"
```

Comment:
In addition to this change, I think we're about due for another release of `password-hash`